pkg/gce-pd-csi-driver/controller.go

```diff
@@ -52,23 +52,38 @@ const (
 	attachableDiskTypePersistent = "PERSISTENT"
 )
 
-func getRequestCapacity(capRange *csi.CapacityRange) (capBytes int64) {
+func getRequestCapacity(capRange *csi.CapacityRange) (int64, error) {
```

Comment:
You may want to check out what I did for this (in fact, it should probably be a utility function that can be shared by multiple drivers), and my test cases.

Reply:
I wrote some tests for this function in a followup PR. Lets work on a canonical `getRequestCapacity`.